src/js/edu-benefits/5490/config/form.js

```diff
@@ -498,11 +502,25 @@ const formConfig = {
                   'ui:options': {
                     hideIf: form => {
                       const status = _.get('highSchool.status', form);
-                      return status !== 'graduated';
+                      return status !== 'graduated' && status !== 'graduationExpected';
                     },
-                    expandUnder: 'status'
-                  }
-                }
+                    expandUnder: 'status',
```

Comment:
The way it's written right now, there's no animation when the fields are shown, I think because of how `hideIf` and `expandUnder` interact. Being able to be more specific about when fields expand underneath the status might help (e.g. have something like `expandUnderValue: 'graduationExpected'`).

I think it's fine for this PR, but we should create an issue to work on it later.